fluent-plugin-events/README.md

```diff
@@ -1,8 +1,42 @@
 # fluent-plugin-events
 
-[Fluentd](https://fluentd.org/) input plugin to do something.
+Use the [Fluentd](https://fluentd.org/) input plugin to watch Kubernetes events from the Kubernetes API Server.
 
-TODO: write description for you plugin.
+### Events
+
+To customize fields for events edit the provided (`fluentd-sumologic.yaml`) file by specifying the fields in the `events` plugin `@type` parameter. The following example has specified the `type_selector`, `deploy_namespace`, and `configmap_update_interval_seconds` fields,
+
+```json
+<source>
+    @type events
+    deploy_namespace sumologic
+    type_selector ["ADDED", "MODIFIED", "DELETED"]
+    configmap_update_interval_seconds 5
+</source>
+```
+
+#### Configurable fields for events
+
+Parameter Name | Default |Description |
+------------ | ------------- | -------------
+resource_name | "events" | Collect events for a specific resource type, such as pods, deployments, or services.
+tag | "kubernetes.*" | Tag collected events.
+namespace | nil | Collect events from a specific namespace.
+label_selector | nil | Collect events for resources matching a specific label.
+field_selector | nil | Collect events for resources matching a specific field.
+type_selector | ["ADDED", "MODIFIED"] | Collect specific event types. Currently supports "ADDED", "MODIFIED", and "DELETED".
+configmap_update_interval_seconds | 10 | Resource version is used to resume events collection from where it left off after a container/pod/node restart. The latest resource version of your events is kept in memory and backed up to a ConfigMap at an interval. By default, we back up the resource version by making a ConfigMap API call every 10 seconds. If you want to back up more frequently, reduce the interval. If you want to reduce the number of API calls, increase the interval.
+watch_interval_seconds | 300 | Interval at which the watch thread gets recreated.
+deploy_namespace | "sumologic" | Namespace that the events plugin resources will be created in. 
+kubernetes_url | nil | URL of the Kubernetes API.
+api_version | v1 | Version of the Kubernetes Events API.
```

Comment:
We should be clear that this `api_version` corresponds to the resource specified by the `resource_name` config parameter. Maybe we can even move this up to line 23, below the `resource_name` since they are coupled